communications.md

```diff
@@ -2,95 +2,62 @@
 
 ## General guidelines
 
-Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+Communication is critical to any organization and in doing so, we should be considerate of other people's production processes and time.
+We use a few tools to aid customer and team (inter and intra) communication: Slack, G Suite, Asana and Emails.
 
-## First line of Communication: Texting
```

Comment:
this _can_ be rephrased, but the original intent of this should remain - we should suggest teams to prefer async communication (written), over synchronous communication (verbal).

Reply:
woozyking in this case it could be opinionated. 

- to **prefer**  async but I honestly don't. Communication is not complete unless there is a feedback and IMO I mostly prefer quick feedback. Feedback could be "I will get back to you", "give me 30 minutes" or otherwise but reasonable. 

We operate an open office to show we welcome everyone to our desk and if not free, we can always politely ask to discuss later hoping the other member of staff will respectfully understand. 

We should bear in mind that there are some people who you can approach in the kitchen or at lunch to converse on the job issues and they don't mind. 
Communication preferences are different that is why I suggested to feel free to find out or ask our pretty sized team what works best for them individually . 
That way we can create an understanding of how to properly communicate with different members of staff based on their preference. 

**My suggestion**
On this async or sync, we can define what it means, state its individual pros and cons and suggest it's usage as deemed appropriate with strong consideration to the other parties preference. Avoid the word default as well

Reply:
This right now is asynchronous communication -- in the world of open-source collaboration, there's not much "I will get back to you" (it's redundant. Members are already expected to respond).

As I mentioned. This has been de-opinionated based on [previous suggestion](https://github.com/EQWorks/common/pull/7#pullrequestreview-323333862), [in this commit](https://github.com/EQWorks/common/pull/7/commits/b9ef34b6b5570bc7d66bd3c2ff7dc533b7cb196c). This is already not just about opinion, but a bottom-line.

Reply:
The open office is not a gesture of welcoming shoulder tapping, unfortunately. This is the truth that interrupts much of our productivity. The capability of context-switch differs from person to person, but it generally holds true that once you "welcome" them, you can't get as much done as you could otherwise.

Reply:
If we keep on the scattered work-related communication at kitchen-sink (or let's say the smoke-session discussions demonstrated in the show "Friends"), then it's already a problem by itself.

By establishing a bottom-line of documenting (written) work-related matters, you avoid the problem of not always go to the kitchen sink (or for Rachael Green in the show, not smoking).

Reply:
> The open office is not a gesture of welcoming shoulder tapping, unfortunately. This is the truth that interrupts much of our productivity. The capability of context-switch differs from person to person, but it generally holds true that once you "welcome" them, you can't get as much done as you could otherwise.

@woozyking 
Waow!! This is what I want us to try to avoid. The first sentence can come across as setting a precedent for an "unwelcoming culture". 
Cheers.

Reply:
Hold the cheers for now. Let’s dive to the point.

If open office is actually a way of showing everyone’s welcome to our desk, then why would some prefer “corners” and even positions adjacent to nobody. Furthermore, most would try to put headphones on to minimize distractions, let alone “welcoming”.

Stop equating a certain office arrangement with how everyone perceives what it means. It’s just an office arrangement. Frankly a suboptimal one to many.

Reply:
I think text is a good starting point for communication. we don't want to scarifies efficiency for friendly, and under high stress situation, tapping on someone's shoulder can spark some unnecessary tension, especially if your involved in multiple project and facing different group, one or two shoulder tappings or "friendly" requests for short conversation from each group can destroy anybody's effectiveness. I understand @odesiuz 's concern on forcing every communication to text can be seen as "unfriendly", but use text to start a communication is absolutely essential. this is also done to respect the person that your communicating, their preference and schedule should always be considered before the conversation start, so asking in slack is a must in my opinion.

Reply:
what about adding this to your slack status message ; whether you're open for people to approach you. when you're deep in a project you can always change it to text first. That way someone who consistently prefers async/text can indicate as such.

The way people work is quite diverse, and like @odesiuz, I flourish a lot with in person interactions (within reason). So what's needed is for people to be able to be respectful of one another's choice and preferences.

How about "indicate your contact preference on your slack status" and then the outreach portion can say "check the person's slack status" before you reach out. A slack note or email is always welcome obviously without checking first.

We can maybe even put this up on a dashboard with who is in; and what their current status is.

Reply:
putting preferred way of communication into slack status is a good idea.